### PR TITLE
chore(release): 9.0.0

### PR DESCRIPTION
**Pull request by joyenjoyer**

### CODE REVIEW

File: docs/contributor-docs/v9-upgrade-guide.md [hunk 1]
```diff
@@ -14,42 +14,48 @@ We recommend upgrading your application for each major version gradually, e.g. i
```
Comment (matyasf) [inline docs/contributor-docs/v9-upgrade-guide.md:14]: This is not a work in progress document (line 9)

File: docs/contributor-docs/v9-upgrade-guide.md [hunk 2]
```diff
@@ -14,42 +14,48 @@ We recommend upgrading your application for each major version gradually, e.g. i
...
+| [ApplyTextDirection](#ApplyTextDirection)     | Use [InstUISettingsProvider](#InstUISettingsProvider) instead to configure text direction.                                                                                |
+| [bidirectional](#bidirectional)               | It has been renamed to [textDirectionContextConsumer](#textDirectionContextConsumer), functionality remains the same.                                                     |
+| [Browser](#Browser)                           | It has been removed in v9.                                                                                                                                                |
+| [ComponentIdentifier](#ComponentIdentifier)   | This util has been removed because it is not used by InstUI and other Instructure projects. If you need it, just copy & paste their code.                                 |
+| [DateTime](#DateTime)                         | This util has been removed because it is not used by InstUI and other Instructure projects. If you need it, just copy & paste their code.                                 |
```
Comment (matyasf) [inline docs/contributor-docs/v9-upgrade-guide.md:42]: seems to be still here

File: docs/contributor-docs/v9-upgrade-guide.md [hunk 3]
```diff
@@ -14,42 +14,48 @@ We recommend upgrading your application for each major version gradually, e.g. i
...
-| [elementMatches](#elementMatches)             | This polyfill is only needed for old, unsupported browsers.                                                                                         |
-| [EmotionThemeProvider](#EmotionThemeProvider) | It has been renamed to [InstUISettingsProvider](#InstUISettingsProvider) and can now also configure text direction.                                 |
-| [isEdge](#isEdge)                             | It has been removed in v9.                                                                                                                          |
-| [isIE11](#isIE11)                             | It has been removed in v9.                                                                                                                          |
+| Utility                                       | Substitute / Notes                                                                                                                                                        |
```
Comment (matyasf) [inline docs/contributor-docs/v9-upgrade-guide.md:35]: We should not link the removed stuff, these just lead to 404 pages

File: docs/contributor-docs/v9-upgrade-guide.md [hunk 4]
```diff
@@ -64,7 +70,7 @@ For more information, click the name of the component to see its full documentat
 ### Removed packages and scripts
 
 - InstUI-CLI's `create-app`, `create-component` and `create-package` commands: Just follow the instructions in the [usage](#usage) docs.
-- The `template-component`, `template-app` and `template-app` packages have been removed.
+- The `template-component`, `template-package` and `template-app` packages have been removed.
 
 ### Removal of deprecated legacy reference attributes from components
```
Comment (matyasf) [inline docs/contributor-docs/v9-upgrade-guide.md:76]: line 77: '...will be deleted in v9'. Isnt this v10? it would be good to list the exact changes here.

**Review by matyasf — CHANGES_REQUESTED** [review]
see my comments, other small stuff that needs to be changed:
- in the opening page of the docs it states '8.0 upgrade guide' and the whats new box links the 8,0 upgrade guide too
- https://1509--preview-instui.netlify.app/#usage adds InstUI `^8` as a dependency
-

**Review by HerrTopi — APPROVED** [review]

**Review by matyasf — CHANGES_REQUESTED** [review]
Another issue: in CodeSandboxButton it will still pull in v8 dependencies:
```
'@instructure/debounce': '^8',
'@instructure/ui': '^8',
'@instructure/ui-icons': '^8',
```
when clicking on the "edit in CodeSandbox"

in docs/usage.md it still says "^8"

**Review by git-nandor — APPROVED** [review]

**Pull request merged**